songnotes/songnotes/settings.py

```diff
@@ -100,6 +103,19 @@
     },
 ]
 
+# Authentication Backends
+# https://docs.djangoproject.com/en/dev/ref/settings/?from=olddocs#authentication-backends
+AUTHENTICATION_BACKENDS = (
+    'social_core.backends.spotify.SpotifyOAuth2',
+    'django.contrib.auth.backends.ModelBackend',
+)
+
+SOCIAL_AUTH_URL_NAMESPACE = 'social'
+
+SOCIAL_AUTH_SPOTIFY_KEY = '8b48dd288bec46eaa1e1c3d06c9ce36d'
+SOCIAL_AUTH_SPOTIFY_SECRET = 'f84aa0c3d8514672a56a6443a574ad94'
```

Comment:
It's a best practice to not put secrets like these into code files. You can replace this line in the settings.py file with: 
```python
import os

SOCIAL_AUTH_SPOTIFY_SECRET = os.environ['SOCIAL_AUTH_SPOTIFY_SECRET']
``` 

Then set the variable using the `export` command before running django.